src/main/java/core/basesyntax/ColorProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorProducer {
+    public static final int RANDOM_COLOR = 10;
+
+    public Color insertRandomColor() {
+        int randomGenerate = new Random().nextInt(RANDOM_COLOR);
```

Comment:
```suggestion
        int colorIndex = new Random().nextInt(Color.values().length);
```
let's use enum length